### DISCUSSION TITLE
Pictures in a zoom speech

### DISCUSSION THREAD

**Discussion by sajaluke** [sircharlo/meeting-media-manager#291] [Q&A] [answered]
Hi, thank you for the wonderful job you are doing! A brother sent me the link of the program and I immediately fell in love with it !!

I notice a problem when I have to transmit an image of a Zoom speech. I use OBS websocket. When the JWMMF monitor is disabled with its button and I am seeing the brother on Zoom, as soon as I send the image the JWMMF monitor returns to the foreground and when I remove the image the writing of the year appears instead of the Zoom speaker.
I think when JWMMF monitor is disabled and I send multimedia content it should remain disabled in order to use the image swapping done by OBS Websocket.

I hope I have explained.

Thank's for your job.
Luca

**Comment by sircharlo** [comment]
Hi there! Glad to hear you're enjoying the app 😄 
I'm afraid I don't quit understand your suggestion though.
You mean that "playing" a picture/video in JWMMF should not automatically show the media window if it was hidden?
My logic was that if the "play" button is pressed on a media file, logically the file is to be shown to the congregation that is at the Kingdom Hall; unless my logic is flawed, or perhaps I misunderstood you?

**Reply by riemsesy** [reply]
it works like intended. 
the problem is, brother in Zoom has a presentation and is shown on the tv/wall.
then the image in JWMMF is shown, of course, the brother disappears, because the picture has to be shown.
then when the image is closed, the brother in zoom needs to be on the front again so he is visible again, but instead, JWMMF shows year text, that's how it's intended.

an option can be to disable the possibility to show year text with JWMMF. We made a screenshot of the year text and use it as background.
or I guess an alt+tab can do that to have the Zoom screen as the focus again.

**Comment by joaochris** [comment, marked as answer]
Isn't it just deactivate the second screen (ALT+Z) display before clicking stop?

https://user-images.githubusercontent.com/76891864/167275378-aebc55f6-a75c-4c06-84b0-14df61e7a4ac.mp4

**Reply by riemsesy** [reply]
Yes looks like it.

**Reply by sircharlo** [reply]
Yeah, using Alt-Z (shortcut for the show/hide button) would be the way to go here, I believe.

**Reply by riemsesy** [reply]
I now understand what he meant with this question. 
Tonight we had this presentation where the video is paused. 
The video stays on screen and in zoom. But the brother giving this presentation asks the question. 
At that moment you can't get him back on screen. Alt+Z results in showing the year scripture. Not the camera. 
Topic starter is right with his question.

**Reply by sircharlo** [reply]
Let me break it down to make sure I understand.

**From KH:**

- Video is playing
- Pause button is pressed, causing the video to be paused
- Alt-Z is pressed, causing the media presentation window to be **hidden** at the KH, revealing the Zoom window underneath on the TV
- The brother asks the question, and the audience answers
- The brother gives the cue to continue the video
- Alt-Z is pressed, causing the media window to be **shown** at the KH, therefore hiding the Zoom window underneath the media presentation window on the TV
- Play button is pressed, causing the video to be resumed

Am I missing something?

**Reply by joaochris** [reply]
I didn't understand 🤔

**Reply by sircharlo** [reply]
You're saying that when Alt-Z is pressed to **hide** the media presentation window, that the chosen OBS camera scene should be triggered, and that when Alt-Z is pressed to **show** the media presentation window, that the chosen OBS media scene should be triggered?

**Reply by riemsesy** [reply]
I guess that is right. 😊

When Alt-z is pressed and a video is paused the cam should be activated for zoom participants. 

Maybe this shouldn't be linked to the Alt-z button because that button has its purpose already. Maybe it should be assigned to the pause button in mp4 media. 

The issue of the topic starter was different because he is talking about a brother in zoom giving a talk. 

Its complex 🤔

**Reply by sircharlo** [reply]
> Maybe this shouldn't be linked to the Alt-z button because that button has its purpose already. Maybe it should be assigned to the pause button in mp4 media.

Good idea! [Done](https://github.com/sircharlo/jw-meeting-media-fetcher/commit/c63b0dd0870641b0b9b0a16a50897efc8ce701f9).
